problemsets-mat223/modules/module10-exercises.tex

```diff
@@ -1,10 +1,52 @@
 \begin{exercises}
+    %45 min
+    %60 min
 	\begin{problist}
-		\prob  Example problem with parts
+	    \prob Let $\mathcal R:\R^2\to\R^2$ be rotation clockwise by $135^\circ$, given by some matrix $M_R$. Compute $M_R$.
```

Comment:
-> `..and let $M_R$ be the corresponding matrix. Find $M_R$`. Change the wording in a similar way for the rest of the questions.